### ISSUE TITLE
Tahmina: Incremental Development

### ISSUE THREAD

**Issue by tahminarasoli** [HackYourFutureBelgium/class-11-12#58]
## Exercises

---

## Week 1

- [ ] Sunday Prep

- [ ] Sunday Class

- [X] [repo](https://github.com/tahminarasoli/acme-web-design) (with a complete README)
- [X] [live demo]( https://tahminarasoli.github.io/acme-web-design/) (turn on GitHub Pages)
- [X] [development strategy]( https://github.com/tahminarasoli/acme-web-design/blob/master/dev-strategy.md)
- [X] [one branch per step]( https://github.com/tahminarasoli/acme-web-design/branches)
- [ ] [one closed PR per step](https://github.com/_/_/pulls)

### Progress
Managing branches
### Blocked
Add particular file in respective branch
### Next Steps
Try again and look for solution to fix the issue :thinking:
### Tip(s) of the week

---

### What went well
Everything is going well even the errors :joy:
### What went less well
I couldn’t realize what mistake I made, suddenly my two pages are not showing up. anyways, I will try to figure it out until end of the week.
### What to do differently next module
Prep study 
### Reflections
The errors I got during this project made me work harder than last week.

**Comment by bermarte** [comment]
Hey Tahmina, 
https://tahminarasoli.github.io/AcmeWebDesign/About.html does not work (**About** written with a capital letter) but https://tahminarasoli.github.io/AcmeWebDesign/about.html it does (**about** written in lowercase).

**Comment by colevandersWands** [comment]
@tahminarasoli , have you been able to figure out the problem with your About and Service pages?

And a small change in your issue: the _Retrospective_ section is for looking back on what you learned in the entire module, so in two weeks from now.  Until then you can leave it empty

**Comment by tahminarasoli** [comment]
yes, thanks to Bernardo Martelli. I was looking for a big mistake but it was a small one :hand_over_mouth:

**Comment by snicoll** [comment]
@tahminarasoli thank for sharing your progress. 

Some initial feedback:

* Your description misses the reference for the team branchies project (pink panther)
* The Acme web design rendering must have a header that allows coaches to validate your CSS and your HTML but that's missing for you
* Your development strategy should have one section per step (per branch) and each of them should be explained in details. I know this sounds a bit silly but the objective here is to get used to decompose a goal into smaller tasks upfront and then execute the plan. The title of each step shouldn't be technical but rather what feature (what added value) you bring to the project
* I've added a few comments on your PRs to guide you a bit more.

Updating this of course doesn't require you to redo all your branches. Please push additional commit to `master` directly so that we can continue the review.

**Comment by iLienstAr** [comment]
Hey @tahminarasoli , this is the one and only individual issue for this module.
You should add your exercises for each week to this issue; and also add the label of the week (for this week: week-2);
And as it is Wednesday today, you should also add the label 'check-in-2' so coaches can check your progress ;)

If it's not clear, let us know. 🙏 
Good luck! 🍀

**Comment by colevandersWands** [comment]
Hey there @tahminarasoli , thanks for checking in!

> Add particular file in respective branch

could you be a little more specific about this?  Maybe share a screen shot or a link to the code that you're blocked on?

**Comment by tahminarasoli** [comment]
@snicoll Hello Stephane, thank you for your comments. @iLienstAr Hi Lien, not really I didn't understand it :( 
@colevandersWands thank you Evan, it was weird. I had to add the index file manually in the url to get the live demo of my project. now it looks fine.
